packages/mediapipe-task-text/lib/src/interface/tasks/text_embedding/text_embedder_result.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:mediapipe_core/interface.dart';
+
+/// {@template TextEmbedderResult}
+/// Container with results of MediaPipe's `embedText` task.
+///
+/// See also:
+///  * [MediaPipe's TextEmbedderResult documentation](https://developers.google.com/mediapipe/api/solutions/java/com/google/mediapipe/tasks/text/textembedder/TextEmbedderResult)
+/// {@endtemplate}
+abstract class BaseTextEmbedderResult extends BaseEmbedderResult {}
```

Comment:
What is this class for?

Reply:
I went back and forth on the value of this class to namespace which type of EmbedderResult we're talking about here, but I gather your vote is to not have this class.

I'll take this out :)